anime-list-master.xml

```diff
@@ -2289,7 +2289,7 @@
   <anime anidbid="551" tvdbid="hentai" defaulttvdbseason="1" episodeoffset="" tmdbid="" imdbid="">
     <name>Choujin Densetsu Urotsukidouji</name>
   </anime>
-  <anime anidbid="552" tvdbid="137801" defaulttvdbseason="0" episodeoffset="" tmdbid="" imdbid="tt0264837">
+  <anime anidbid="552" tvdbid="" defaulttvdbseason="0" episodeoffset="" tmdbid="" imdbid="tt0264837">
```

Comment:
This one could be set as `movie` instead since the name indicates it is one and has `imdbid` set